pkg/rancher-desktop/assets/styles/vendor/vue-js-modal.scss

```diff
@@ -1,12 +1,16 @@
-@import '~/../../node_modules/vue-js-modal/dist/styles.css';
+// NOTE: Commenting out everything in this Sass file for the time being. Rancher 
```

Comment:
Wouldn't using `/*` `*/` be better then?

Reply:
I've updated to make use of sass multiline comments.